src/editors/editor.ts

```diff
@@ -56,6 +58,45 @@ export default abstract class Editor implements EditorInterface {
     }
   }
 
+  public async installJetbrainsPlugin(pluginsDirectory: string): Promise<void> {
```

Comment:
It's not a good idea since this method is not common for all the editors. Need to find a way of making it generic as possible and with no dependency from `Editor` class.

Reply:
yes i was thinking maybe put this logic in the utils folder, what do you think?

Reply:
that's make more sense.